force-app/main/default/classes/CRM_JournalUtilities.cls

```diff
@@ -42,59 +42,27 @@ public class CRM_JournalUtilities {
         CRM_JournalService service = new CRM_JournalService();
         UtilityResponse resp = new UtilityResponse();
 
-        Boolean complete = true; //Removed BID specific handling as a results of FAGSYSTEM-189315
```

Comment:
Æ likte faktisk veldig godt denna kommentaren ngl. Æ huske at når æ først så på Journalførings APIet og satt mæ faktisk inn i det, så va æ overaska over den `forsoekFerdigstill` delen, men denna ga en veldig god forklaring for koffer det va en ting. Æ tenke at enten så fjærne vi hele funksjonaliteten fra CRM_JournalService, eller så beholder vi kommentaren.